test/testdata/godox.go

```diff
@@ -0,0 +1,7 @@
+//args: -Egodox
+//config: linters-settings.godox.keywords=FIXME,TODO
+package testdata
+
+func todoLeftInCode() {
+	// TODO implement me // ERROR godox.go:6: Line contains FIXME/TODO: "TODO implement me"
```

Comment:
please, add more tests:
```
//TODO no space
// TODO(author): 123
//TODO(author): 123
//TODO(author) 456
// TODO: qwerty
// todo 789
```

Reply:
Added. My reasoning was that tests for all these cases are in the repository with `godox` and as such are not needed here.